Frameworks/include/Starboard.h

```diff
@@ -205,4 +205,19 @@ inline CAPoint3D operator*(float f, const CAPoint3D& v) {
     return CAPoint3D(v.x * f, v.y * f, v.z * f);
 }
 
+// Wrapper for C++/WinRT types when you need a trivial default constructor
```

Comment:
C++/WinRT does provide default constructors, but they have behavior that can be surprising. The default constructor will instantiate the underlying object, unlike other smart pointer types like ComPtr or unique_ptr. This wrapper is provided for those cases where you need to have a default constructor that doesn't  instantiate the underlying object.

---
In reply to: [109485708](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109485708) [](ancestors = 109485708)